perses/rjmc/geometry.py

```diff
@@ -474,30 +485,8 @@ def _angle_logp(self, theta, angle, beta):
         """
         k_eq = angle.type.k*units.kilocalories_per_mole/(units.radians**2)
```

Comment:
The AMBER force constants may also be off by a factor of two.  Trace the call @swails uses here carefully to double-check:
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2053-L2054